### PR TITLE
Document the now_show tactic.

### PR DESCRIPTION
**Pull request by Zimmi48**
It was used in the standard library and the test-suite but undocumented so far.

**Kind:** documentation

### CODE REVIEW

File: doc/sphinx/proof-engine/tactics.rst
Comment (cpitclaudel) [inline doc/sphinx/proof-engine/tactics.rst]: typo: a synonym of

**Review by cpitclaudel — CHANGES_REQUESTED** [review]

File: doc/sphinx/proof-engine/tactics.rst
Comment (Zimmi48) [reply doc/sphinx/proof-engine/tactics.rst]: Thanks. Fixed!

**Comment by cpitclaudel** [comment]
Perfect, thanks! Quick question: when I merge, the merge commit adds an `Ack-By` line on the merge commit, with the commiter ID of the original commit; does that look OK? An `Ack-By` on the original commit (with the merger's ID) would seem more logical.

```
commit e3d1a567290ec718a31622177d54f15f4cd5e8c8
Merge: 7831c25 57c69e8
Author: Clément Pit-Claudel <clement.pitclaudel@live.com>
Date:   Thu Feb 14 10:29:12 2019 -0500

    Merge PR #9571: Document the now_show tactic.
    
    Ack-by: Zimmi48

commit 57c69e8b78df99e69a31c0a6129346915de5e38c
Author: Théo Zimmermann <theo.zimmermann@univ-paris-diderot.fr>
Date:   Wed Feb 13 18:33:46 2019 +0100

    Document the now_show tactic.
    
    It was used in the standard library and the test-suite but undocumented so far.
```

**Comment by SkySkimmer** [comment]
Github says @Zimmi48 commented so the script adds the Ack-by.
BTW you should approve before merging ;)

**Comment by cpitclaudel** [comment]
Got it, thanks for clarifying. And thanks for the merge-process tip, too :)

**Review by cpitclaudel — APPROVED** [review]

**Comment by Zimmi48** [comment]
I usually remove the Ack-by line corresponding to the author. I do not find that this adds much useful information unless the PR was taken over by someone else.

**Pull request merged**
